src/api/storage.py

```diff
@@ -354,11 +456,17 @@ def sliding_window_test(self, regressor, sliding_window, prior):
                         else:
                             # Use previous sample to shift window
                             self._data.loc[index][col_labels[2:]
-                                                  ] = list(prev_sample.iloc[-2*sliding_window:-2])
```

Comment:
Move first closing square bracket to previous row, and break after equals sign instead (use backslash to allow this line break). There is no need to have all of this blank space - fix this if there are more areas where it looks oddly spaced like this (I believe the same things happens at like 3 different places as well).